OpenQA/Isotovideo/Utils.pm

```diff
@@ -12,15 +12,17 @@ use bmwqemu;
 use autotest;
 use Try::Tiny;
 
-our @EXPORT_OK = qw(checkout_git_repo_and_branch checkout_git_refspec
-  handle_generated_assets load_test_schedule);
+our @EXPORT_OK = qw(git_rev_parse checkout_git_repo_and_branch
+  checkout_git_refspec handle_generated_assets load_test_schedule);
+
+sub git_rev_parse ($dirname) {
+    my $version = (-e "$dirname/.git" ? qx{(git config --global --get safe.directory | grep -q $dirname || git config --global --add safe.directory $dirname) && git -C $dirname rev-parse HEAD} : '') || 'UNKNOWN';
```

Comment:
This needs write-permissions but normally we have `_openqa-worker` running with a home directory /var/lib/empty which does not offer write permissions for _openqa-worker. Not sure what to do instead.

Reply:
Rewritten to read out the git ref manually avoiding git commands.